### PR TITLE
test(java): Add test for ClassLoaderFuryPooled

### PR DESCRIPTION
**Pull request by LiangliangSui**
Add test for `ClassLoaderFuryPooled`.

### CODE REVIEW

**Comment by LiangliangSui** [comment]
@chaokunyang ptal, thanks!

File: java/fury-core/src/test/java/org/apache/fury/pool/ClassLoaderFuryPooledTest.java [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+  @Test
+  public void testReturnFury() {
+    try {
+      Function<ClassLoader, Fury> furyFactory = getFuryFactory();
+      Fury fury = furyFactory.apply(getClass().getClassLoader());
+      ClassLoaderFuryPooled pooled = getPooled(4, 8, furyFactory);
+      pooled.returnFury(fury);
+    } catch (RuntimeException e) {
+      Assert.fail();
```
Comment (chaokunyang) [inline java/fury-core/src/test/java/org/apache/fury/pool/ClassLoaderFuryPooledTest.java:122]: Seems we can just remove catch block

Comment (LiangliangSui) [reply java/fury-core/src/test/java/org/apache/fury/pool/ClassLoaderFuryPooledTest.java]: Updated

**Review by chaokunyang — APPROVED** [review]
LGTM

**Pull request merged**